lms/templates/certificates/accomplishment-base.html

```diff
@@ -0,0 +1,40 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! import mako.runtime %>
+<% mako.runtime.UNDEFINED = '' %>
+
+<%
+# set doc language direction
+from django.utils.translation import get_language_bidi
+dir_rtl = 'rtl' if get_language_bidi() else 'ltr'
+document_body_class = document_body_class_append if document_body_class_append else ''
+%>
+
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+<head dir=-"${dir_rtl}">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>${document_title}</title>
+
+    <%include file="_assets-primary.html" />
+</head>
+
+<body class="layout-accomplishment view-valid-accomplishment ${dir_rtl} ${document_body_class} certificate certificate-distinguished" data-view="valid-accomplishment">
```

Comment:
NOTE: @mattdrayer, we'll need to sync up the following classes here:

* ``certificate`` - added to all accomplishments that are certificates
* ``certificate`` + ``certificate-base`` - base certificate
* ``certificate`` + ``certificate-distinguished`` - distinguished certificate
* ``view-valid-accomplishment`` - if accomplishment is valid
* ``view-invalid-accomplishment`` - if accomplishment is invalid (we'll be showing the invalid.html file with specific UI here too)